src/me/actions/thunks/index.ts

```diff
@@ -35,6 +36,10 @@ export const getMe = () => async dispatch => {
       user_id: user.id,
     })
 
+    if (isFlagEnabled('rudderStackReporting')) {
+      identify(user.id, {email: user.name, orgID: this.props.orgID})
```

Comment:
where is orgID coming from?

Reply:
Also, should we stick with checking this only in CLOUD?

Reply:
Updated to get orgID properly and added a check for CLOUD.